### DISCUSSION TITLE
Request Feature - Mass Handing of Expired Articles

### DISCUSSION THREAD

**Discussion by Cpointcc** [joomla/joomla-cms#47152] [2. Cold Features]
### Steps to reproduce the issue

Current Joomla articles Select Status - No option to select/filter expired articles

### Expected result

To be able to archive, delete or unpublish expired articles in a mass function.

### Actual result
No Option to choose unpublished articles except manually

### System information (as much as possible)

Joomla 3.9.20 - Not an option

### Additional comments

This would save me a lot of time manually managing expired content.

**Comment by brianteeman** [comment]
What I do is to sort by finished publishing
![image](https://user-images.githubusercontent.com/1296369/90964536-5850e280-e4b9-11ea-9d1e-2e2da6c4c091.png)

**Comment by Cpointcc** [comment]
That's an option but I still would have to manually select them to batch file anything.  Thanks for you input though.

**Comment by Sukinoz** [comment]
Hello,

Same problem. 

**Sort by "finish publisihing descending" makes:**

1. - First it shows all articles with end of publication date after today.
2. - Then all expired articles (the ones we want to filter).
3. - And then all articles with no end date.

So, you do not know where beging and where end the expired articles in the list.

PS: with more than 5000 articles to manage, this is not the most comfortable
PPD:  Just a comment. Editors often ask me confused: why in Joomla when the "end of publication date" is reached, the article does not become "unpublished"? I tell them that they become "expired", but they insist that "expired" is the reason why it is not published, but not the status.

**Comment by davidascherG** [comment]
It seems clear that a published article that has passed its publish_down date is treated by the article manager as "published" article, although the icon to indicate that the article has passed its "expiration" date is displayed. I seem to recall that way back in Joomla 1 or 1.5, there was an option either via some long gone extension or in the core that would change an article's state to "archived" under these conditions. "archived" a state indicated by the value of the article's "access" field - along with "published", "unpublished", and "trashed". It seems long past time for "expired" to be treated as an additional state by the article manager.  Many site content managers have lost lots of hair over the past decade or so trying to locate the "expired" articles to move them to some different category or archive them. It doesn't seem like brain surgery level coding would be required to implement this, but perhaps I am wrong. OR perhaps this issue has been taken care of in Joomla 4 - which I have not yet had a chance to peruse.

**Comment by denvarel** [comment]
## Up!
This limitation can be challenging in practice. When articles reach their expiration date, they become invisible. There is currently no way to filter expired articles, and it is also difficult to archive them or move them to another category unless the expiration date is adjusted; otherwise, the articles remain invisible.

It would be very helpful to have a way to mass-handle expired articles, such as archiving them or moving them to a different category, while also allowing the expiration date to be modified (for example, cleared or extended). Without the ability to adjust the expiration date, managing expired content becomes unnecessarily restrictive.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/30446">issues.joomla.org/tracker/joomla-cms/30446</a>.</sub>

**Comment by brianteeman** [comment]
This task plugin will archive all expired articles https://github.com/conseilgouz/plg_task_cgautoarchive_j4
